src/App.css

```diff
@@ -42,59 +37,67 @@
   background-color: #252e38;
   box-shadow: 0 0 25px 0 #131c25;
 }
-#navigation .navigation_nav-container {
+#navigation .navigation-nav-container {
+  float: right;
   display: flex;
   justify-content: center;
   align-items: center;
   height: 100%;
   padding: 0 20px;
 }
-#navigation .navigation_nav-container {
+#navigation .navigation-nav-container {
   display: flex;
   justify-content: center;
-  align-items: center;
   height: 100%;
   padding: 0 20px;
 }
 .navigation-logo {
-  margin: 0 10px;
+  margin: 10px 10px;
   height: 100%;
-  display: flex;
   align-items: center;
-  transition: opacity,.2s;
-  /* background-image: url(https://static.codingame.com/assets/spritesheets/navigation.6e3a4d9e.png); */
-  width: 138px;
+  transition: opacity, 0.2s;
   height: 18px;
   width: 142px;
+  padding-top: 30px;
+  font-size: 250%;
+  color: white;
+  font-display: bold;
+  font-family: fantasy;
 }
 .navigation-auth {
   display: flex;
-  margin: 0 15px;
+  margin: 6px 15px;
   flex-basis: 310px;
+  float: right;
+  padding: 4px;
+  width:170px
 }
-.navigation-auth .navigation-auth_login-button {
+.navigation-auth .navigation-auth-login-button {
   text-align: center;
-  background-color: hsla(0,0%,100%,.08);
+  background-color: hsla(0, 0%, 100%, 0.08);
   color: #fff;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
font-size make it 14 like the singup button

Reply:
I've changed its font size to 14px